pkg/driver/deployment_dandelion.go

```diff
@@ -0,0 +1,67 @@
+package driver
+
+import (
+	"fmt"
+	"github.com/sirupsen/logrus"
+	"github.com/vhive-serverless/loader/pkg/common"
+	"io"
+	"math/rand"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+type RegisterFunction struct {
+	Name        string  `bson:"name"`
+	ContextSize uint64  `bson:"context_size"`
+	EngineType  string  `bson:"engine_type"`
+	Binary      []int32 `bson:"binary"`
+}
+
+func DeployFunctionsDandelion(controlPlaneAddress string, functions []*common.Function) {
```

Comment:
Please deduplicate code and add Dandelion-specific inside If statements inside the existing `deployDirigent` function. I think registration can be put into the same file as for regular Dirigent.